source/dotnet/Library/AdaptiveCards.Rendering.Wpf.Xceed/XceedAdaptiveCardRenderer.cs

```diff
@@ -7,9 +7,17 @@ public static class XceedAdaptiveCardRenderer
         public static void UseXceedElementRenderers(this AdaptiveCardRenderer renderer)
         {
             renderer.ElementRenderers.Set<AdaptiveTextInput>(XceedTextInput.Render);
-            renderer.ElementRenderers.Set<AdaptiveCards.AdaptiveNumberInput>(XceedNumberInput.Render);
+            renderer.ElementRenderers.Set<AdaptiveNumberInput>(XceedNumberInput.Render);
             renderer.ElementRenderers.Set<AdaptiveDateInput>(XceedDateInput.Render);
             renderer.ElementRenderers.Set<AdaptiveTimeInput>(XceedTimeInput.Render);
         }
+
+        public static void DontUseXceedElementRenderers(this AdaptiveCardRenderer renderer)
```

Comment:
>DontUseXceedElementRenderers [](start = 27, length = 28)

[nit] change the name of this method to something like "UseWpfInputElementRenderers" or "UseNonXceedElementRenderers", something about what it is doing rather than what it is not doing

Reply:
Good call. UseDefaultElementRenderers?

---
In reply to: [312667350](https://github.com/microsoft/AdaptiveCards/pull/3326#discussion_r312667350) [](ancestors = 312667350)